actions/new.action.ts

```diff
@@ -250,4 +255,14 @@ export const retrieveCols = () => {
   }
 };
 
+const fileExists = (path: string) => {
+  return promisify(fs.access)(path).then(() => true).catch((err: any) => {
```

Comment:
actually, I don't see any reason to use promises here because all the other tasks will need to wait the `createGitIgnoreFile` one

So can you change `fs.access` with [`fs.accessSync`](https://nodejs.org/api/fs.html#fsaccesssyncpath-mode)?

Reply:
Yes

Reply:
Just changed the flow to use the synchronous access and mainly reverted the createGitIgnoreFile to be mostly the same code except adding the check to file exist then it returns.

Reply:
can you run `npm ci && npm run format` to fix formatting issues?

Reply:
but only commit the `actions/new.action.ts` xD

Reply:
nvm, let me help you with that. I just ran that command myself